### DISCUSSION TITLE
[Cloud Engineer] Use software patterns and architectures that best support data access and storage patterns

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#38] [Ideas]
https://docs.aws.amazon.com/wellarchitected/latest/sustainability-pillar/use-software-patterns-and-architectures-that-best-support-data-access-and-storage-patterns.html

Intent: Understand how data is used within your workload, consumed by your users, transferred, and stored. Select technologies to minimize data processing and storage requirements.

**Comment by holanita** [comment]
[Issue created](https://github.com/Green-Software-Foundation/green-software-patterns/issues/53)
